cooluri/link.Functions.php

```diff
@@ -259,7 +259,7 @@ public static function pageNotFound($lConf) {
       if (!empty($lConf->cache->pagenotfound)) {
           switch ((string)$lConf->cache->pagenotfound->behavior['type']) {
               case 'message': $res = (string)$lConf->cache->pagenotfound->behavior; break;
-              case 'page':    $res = file_get_contents((string)$lConf->cache->pagenotfound->behavior); break;
+              case 'page':    $res = t3lib_div::getUrl((string)$lConf->cache->pagenotfound->behavior); break;
```

Comment:
I don't want dependency on TYPO3 classes in this class. CoolUri core is independent on TYPO3. And this is not compatible with 6.2+